components/call_button/call_button.jsx

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import {injectIntl} from 'react-intl';
+
+import {intlShape} from 'utils/react_intl';
+
+import MenuWrapper from 'components/widgets/menu/menu_wrapper';
+import Menu from 'components/widgets/menu/menu';
+import CameraIcon from 'components/widgets/icons/camera_icon';
+
+const customStyles = {
+    left: 'inherit',
+    right: 0,
+    bottom: '100%',
+    top: 'auto',
+};
+
+class CallButton extends PureComponent {
+    static propTypes = {
```

Comment:
Would it be too much effort to convert this file to TypeScript and specify the prop types using that instead of `propTypes`?

Reply:
In order to properly move this to Typescript, we would need plugin folder migrated to typescript too, to have a clear and centralized typing for the pluginCallMethods. I could type them here, but I do not think it is the best place to do it.